windows-02.md

```diff
@@ -104,7 +86,7 @@ You should get this in return:
 
 And it will download a bunch of files into your folder.
 
-- Run `npm install`. This will also download a bunch of files. It might take a couple of minutes to run.
+- Run `npm ci`. This will also download a bunch of files. It might take a couple of minutes to run.
```

Comment:
> npm ci just installs existing dependencies, in contrast to npm install, which attempts to update current dependencies if possible. This ensures that the builds in continuous integration are reliable. It’s better to use npm i in development and npm ci for production.

[Link](https://levelup.gitconnected.com/npm-i-vs-npm-ci-install-node-modules-in-your-app-faster-and-wisely-e5b1bef0f93d#:~:text=npm%20ci%20just%20installs%20existing,and%20npm%20ci%20for%20production.)